<launch>
  <include file="$(find ros_study)/launch/move_base.launch" />
  <include file="$(find ros_study)/launch/map_server.launch" />
  <node name="vehicle_emu" pkg="ros_study" type="vehicle_emu.py" clear_params="true">
    <param name="publish_twist" value="false"/>
    <param name="publish_imu" value="false"/>
    <param name="publish_path" value="false"/>
    <param name="publish_fix" value="false"/>
  </node>
</launch>
